types/jsonfile/utils/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import { JFReadOptions } from '../';
+import { JFWriteOptions } from '../';
 
-export function stringify(obj: any, options?: JFReadOptions): string;
+export function stringify(obj: any, options?: JFWriteOptions): string;
```

Comment:
```suggestion
export function stringify(
    obj: any,
    options?: {
        EOL?: string | undefined;
        finalEOL?: boolean | undefined;
        replacer?: ((key: string, value: any) => any) | undefined;
        spaces?: string | number | undefined;
    },
): string;
```
it looks like neither of both, just those

Reply:
if you accept this, bump version to 6.1:
https://github.com/jprichardson/node-jsonfile/compare/6.0.1...6.1.0